README.md

```diff
@@ -52,7 +54,7 @@ to configure your IoT-LAB SSH access in this
 
 ### Examples:
 
-* Start an experiment, wait for it ot be ready, wait for all A8 boot
+#### Start an experiment, wait for it ot be ready, wait for all A8 boot:
```

Comment:
ot be ready => **to** be ready

The typo was already there but as your are at it, you can change it.